### ISSUE TITLE
[Docs]

### ISSUE THREAD

**Issue by tidwelljohn01** [microsoft/botframework-solutions#3787]
/solution-accelerators/tutorials/view-analytics/3-open-template/

I have a Virtual Assistant running in Azure Government Cloud, which does not use Azure AD Connect. My user account is a Basic user. I am unable to pull the Applications Insights data into the Power BI template after entering the Application ID. I basically get a 404 error and from the Azure Gov side I can see that the  Applications Insights API has never been hit. What do I need in order to get this to work?

**Comment by lauren-mills** [comment]
Hi @tidwelljohn01 I found this [StackOverflow issue](https://stackoverflow.com/questions/60743120/telemetry-data-not-being-received-by-application-insights-from-azure-functions-o) that seems somewhat similar that indicates the endpoint used in the template at the link above might not be correctly configured for Azure Gov. I'm working to check the template, but it looks like you'll need to change the data import to use applicationinsights.us.

I'll follow up with more instructions once I check the template.

**Comment by lauren-mills** [comment]
I'm having difficulties creating you a new template without a working Azure Gov appInsights of my own, so I'm going to try and walk you through it.

1. Open the .pbit from the link above
2. Instead of entering your API key, click the dropdown by the Load button and click Edit
3. Click the Transform Data in the Home menu
4. Enter your API key in the input field:        
     ![image](https://user-images.githubusercontent.com/27976702/114953468-4982bb80-9e0d-11eb-81f6-36e70208e28e.png)
5. Right-click Event Telemetry and select Advanced Editor
6. Change "https://api.loganalytics.io/v1/apps/" to "https://api.loganalytics.us/v1/apps/"
    ![image](https://user-images.githubusercontent.com/27976702/114953618-9a92af80-9e0d-11eb-903c-a5896fc9fb16.png)
7. Click Close and Apply

I think you should see your data start loading then. Let me know how it goes.

**Comment by GeoffCoxMSFT** [comment]
@tidwelljohn01 Is your issue resolved?

**Comment by tidwelljohn01** [comment]
Sorry, just saw the instruction above. I will try and let you know.

**Comment by tidwelljohn01** [comment]
Progress.

I get a new error. I think the issue is my account type, which is a basic account:

![image](https://user-images.githubusercontent.com/70174577/116136800-6b473280-a698-11eb-8b9b-828c9780a7cf.png)

**Comment by tdurnford** [comment]
@lauren-mills Any updates on this?

**Comment by lauren-mills** [comment]
@tidwelljohn01 - are you able to view the data in the Azure Portal? If so, you should be able to access it in the PBI template.

After changing the url to point to api.loganalytics.us, can you verify you are logging in with your organizational account and it is the same one you use to access the Azure Portal?

If you don't have access in the Azure Portal, then you will need to be granted at least Read access to the App Insights resource in order to load the data in.

**Comment by tidwelljohn01** [comment]
I got it! Thanks so much @lauren-mills. 

The client had told me that my account was a "Basic Account", so I had just been selecting that as the authentication method to the API. But I was able to view the data in the App Insight logs in the Azure portal when logged in with that account, as you asked about. So I selected Organizational Account and entered my username and password, and that was part of the fix. However, I still got this error:

![image](https://user-images.githubusercontent.com/70174577/118713513-a34d1b80-b7e7-11eb-86d0-4a9f52de668c.png)

So I followed your instructions above about changing from .io to .us for the EventTelemtry, and applied that same process to Feedback by right clicking and going into Advanced Editor. And when I applied the changes it connected and pulled "some" data. I think I need more Telemetry in my bot... So I am working on that now.
